src/sim.rs

```diff
@@ -148,6 +150,11 @@ impl<'a> Sim<'a> {
         self.world.borrow_mut().lookup(addr)
     }
 
+    /// Lookup an ip address by host name.
```

Comment:
Comment is singular and code is plural, but also see comment above about only having the regex path.